app/models/project_step.rb

```diff
@@ -196,6 +206,67 @@ def week_of_scheduled_date
     end
   end
 
+  # todo: confirm if we want to hardcode a limit for end_date bound repeats. and if so, what should that be?
+  MAX_OCCURRENCES = 100
```

Comment:
I would say 60. 100 seems like a lot at one time. I worry about performance degrading. Please put constants at top of file.